### ISSUE TITLE
[Bug Report] [Bug] v-combobox cannot select by Enter key

### ISSUE THREAD

**Issue by bob-lee** [vuetifyjs/vuetify#8874]
### Environment
**Vuetify Version:** 2.0.1
**Vue Version:** 2.6.0
**Browsers:** Chromium 76.0.3809.100
**OS:** Ubuntu undefined

### Steps to reproduce
On desktop, try to focus it to open options list then select by `Enter` key. Have to use mouse click to select. Found the same issue [#5008](https://github.com/vuetifyjs/vuetify/issues/5008) which had been closed as resolved. 

### Expected Behavior
Enter key selects the option.

### Actual Behavior
Enter key doesn't select option, it just dismiss the options list.

### Reproduction Link
<a href="https://codepen.io/bob-lee/pen/qBWVQrd" target="_blank">https://codepen.io/bob-lee/pen/qBWVQrd</a>

### Other comments
On ios Chrome, I can select by touch.

**Comment by sh7dm** [comment]
cleaned up the reproduction: https://codepen.io/sh7dm/pen/eYOVmKJ

**Comment by jacekkarczmarczyk** [comment]
Already fixed https://codepen.io/jkarczm/pen/yLBvyqy
